README.md

```diff
@@ -162,26 +180,66 @@ The fragments can be used in other languages too:
 
 ### Configuration
 
-By default, the command recursively reads all files from `_samples` directory and embeds them into
-Markdown files found in `./docs` directory.
+In order for the command to work, you need to specify at least two configurations params:
+ - `code_root` — the directory where all the embedded code resides;
+ - `documentation_root` — the directory where all the docs which need embedding reside.
 
-You can change these settings in the very place they're defined: [../_config.yml]()
+Those parameters should be specified via the Jekyll `_condig.yml` file.
 
 For example:
 ```yaml
 embed_code:
-  code_root: ./_samples                   # The directory that will be recursively scanned for sample code files.
-  code_includes: "**/*.java,**/*.gradle"  # The rules defining which code files to consider.
-  documentation_root: ./docs              # The directory with the Markdown to be processed.
-  fragments_dir: .fragments               # The directory where intermediary results of the plug-in are written.
-  interlayer: "..."                       # A piece of text to be inserted between occurrences of the same fragment.
+  code_root: ./_samples
+  documentation_root: ./docs
+
+  # Optional params:
+  code_includes: ["**/*.java", "**/*.gradle"]
+  doc_includes:  ["docs/*.md", "docs/*.html"]
+  fragments_dir: ".fragments"
+  interlayer: "..."
 ```
 
+Other command configuration parameters include:
+ - `code_includes` — a list of glob patterns defining which code files to consider. By default,
+   all files (`**/*`).
+ - `doc_includes` — a list of glob patterns defining which doc files to consider. By default,
+   Markdown and HTML files (`**/*.md`, `**/*.html`).
+ - `fragments_dir` — a temporary directory for fragment files. The command extracts the code
+   fragments files and stores them in a temporary dir. Consider adding this directory to
+   `.gitignore`. By default, `./.fragments`
+ - `interlayer` — a string which separates partitions of a single fragment in the resulting embedded
+   code. See [fragment doc](#more-on-fragments) for more. The interlayer is automatically appended
+   with a new line symbol. By default, `...`.
+
+## Development
+
+This software is written in Ruby as it's a plug-in for Ruby-based [Jekyll](https://jekyllrb.com/).
+While Ruby may not be familiar to you, this plug-in consists of the trivial string and file
```

Comment:
This section should speak of a development environment, such as Ruby version, Jekyll version etc. Let's not try to calm down the Java developer who reads this file and panics :)